bears/python/ChecksSelector/ChecksSelector.py

```diff
@@ -0,0 +1,23 @@
+from coalib.settings.Setting import typed_list
+
+
+def set_checks(ignore: str = '--ignore',
+               ignore_check: typed_list(str) = (),
+               select: str = '--select',
```

Comment:
`ignore` & `select` are confusing names. `ignore_cli_arg` and `select_cli_arg` are more clear.